x/bonds/alias.go

```diff
@@ -1,45 +1,294 @@
 package bonds
 
+// nolint
+// autogenerated code using github.com/haasted/alias-generator.
+// based on functionality in github.com/rigelrozanski/multitool
```

Comment:
Not sure if we should be generating the entire list of aliases, even those which we do not use? I guess it doesn't hurt to have them. Also did not know about this tool! :grin: